src/System.Text.Json.Dynamic/System/Text/Json/JsonDynamicObject.cs

```diff
@@ -167,8 +177,9 @@ public override bool TrySetMember(SetMemberBinder binder, object value)
         public bool TryFormat(Span<byte> buffer, out int written, TextFormat format, EncodingData formattingData)
         {
             written = 0;
+            int consumed;
             int justWritten;
-            if(!formattingData.TextEncoder.TryEncode('{', buffer, out justWritten)) {
+            if(!formattingData.TextEncoder.TryEncode(OpenCurlySpan, buffer, out consumed, out justWritten)) {
```

Comment:
There is no work item for this at the moment. However, I think it is best we see some real scenarios first and determine if it is necessary. Is it possible to implement a high performance JSON processor without a single-character method?

We can always add APIs, but it is much harder to remove them.